apps/web/pages/auth/login.tsx

```diff
@@ -36,6 +37,12 @@ interface LoginValues {
   csrfToken: string;
 }
 
+const querySchema = z.object({
+  sso: z.string().optional(),
+  sig: z.string().optional(),
+  callbackUrl: z.string().optional().default(""),
+});
+
```

Comment:
Converting already existing query parameters into Zod schema, adding `sso` and `sig` for Discourse.